images/devcontainer/default.nix

```diff
@@ -94,23 +128,19 @@ let
       # make sure /tmp exists
       mkdir -m 0777 tmp
 
-      # allow ubuntu ELF binaries to run. VSCode copies it's own.
-      mkdir -p lib64
-      ln -s ${glibc}/lib64/ld-linux-x86-64.so.2 lib64/ld-linux-x86-64.so.2
-
       # VSCode assumes that /sbin/ip exists
       mkdir sbin
       ln -s /nix/var/nix/profiles/default/bin/ip sbin/ip
     '';
 
     config = {
-      Cmd = [ "/nix/var/nix/profiles/default/bin/bash" ];
+      Entrypoint = [ "/nix/var/nix/profiles/default/bin/entrypoint.sh" ];
```

Comment:
When I was hacking on it, vscode was running the container with its own entry point. Is that not the case anymore?

Reply:
Microsoft does seem to do the same in some of their devcontainers: https://github.com/microsoft/vscode-dev-containers/blob/560deca1511296d64d31acea1ba08feea3f7850b/containers/docker-from-docker/.devcontainer/Dockerfile#L26

Reply:
I mean it works for me if using docker-compose, but have to still test with plain devcontainer.